pkg/loadbalancer/experimental/lbmaps.go

```diff
@@ -242,6 +392,21 @@ func (r *BPFLBMaps) Stop(cell.HookContext) error {
 	return errors.Join(errs...)
 }
 
+// iterateMap iterates over a BPF map, yielding new keys and values. Similar to
+// ebpf.IterateWithCallback but allocates new key & value instead of reusing.
```

Comment:
What's the motivation? Is this lack of reusing a safeguard for the case when cb is non-blocking?

Reply:
`ebpf.IterateWithCallback` reused the same allocated values which means we can't use it for snapshotting maps.